tests/python/test_arrays.py

```diff
@@ -0,0 +1,1024 @@
+    items = [1,2,3]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.copyWithin(0)}")(result, items)
+    assert items == [1,2,3]  
+
+def test_copyWithin_target_only():
+    items = [1,2,3]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.copyWithin(1)}")(result, items)
+    assert items == [1,1,2]     
+
+def test_copyWithin_negative_target_only():
+    items = [1,2,3]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.copyWithin(-1)}")(result, items)
+    assert items == [1,2,1] 
+
+def test_copyWithin_target_too_large():
+    items = [1,2,3]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.copyWithin(10)}")(result, items)
+    assert items == [1,2,3]     
+
+def test_copyWithin_target_and_start():
+    items = [1,2,3]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.copyWithin(1, 2)}")(result, items)
+    assert items == [1,3,3]   
+
+def test_copyWithin_target_and_start_too_large():
+    items = [1,2,3]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.copyWithin(1, 2)}")(result, items)
+    assert items == [1,3,3]       
+
+def test_copyWithin_target_and_negative_start():
+    items = [1,2,3]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.copyWithin(1, -1)}")(result, items)
+    assert items == [1,3,3]   
+
+def test_copyWithin_target_and_start_and_end():
+    items = [1,2,3,4,5]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.copyWithin(1,2,3)}")(result, items)
+    assert items == [1,3,3,4,5]    
+
+def test_copyWithin_target_and_start_and_negative_end():
+    items = [1,2,3,4,5]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.copyWithin(1,2,-2)}")(result, items)
+    assert items == [1,3,3,4,5]    
+
+#includes
+def test_includes():
+    items = [1,2,3]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.includes(1)}")(result, items)
+    assert result[0] == True   
+
+def test_includes_other_type():
+    items = [1,2,'Hi']
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.includes('Hi')}")(result, items)
+    assert result[0] == True         
+
+def test_includes_not():
+    items = [1,2,3]
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.includes(5)}")(result, items)
+    assert result[0] == False   
+
+def test_includes_not_other_type():
+    items = [1,2,'Hi']
+    result = [None]
+    pm.eval("(result, arr) => {result[0] = arr.includes('Joe')}")(result, items)
+    assert result[0] == False   
+
+def test_includes_too_few_args():
+    items = [4,2,6,7]
+    try:
+        pm.eval("(arr) => {arr.includes()}")(items)      
+        assert (False)
+    except Exception as e:    
+        assert str(type(e)) == "<class 'pythonmonkey.SpiderMonkeyError'>"
+        assert str(e).__contains__("TypeError: includes: At least 1 argument required, but only 0 passed")   
+
+
+#sort
```

Comment:
Include a test for `.sort()` that ensures that when no arguments are passed, the default behaviour is to convert the elements to strings and compare them, like so:
```py
def test_sort_numbers_tricky():
    items = [1, 30, 4, 21, 100000]
    result = [None]
    pm.eval("(result, arr) => {result[0] = arr.sort()}")(result, items)
    assert result[0] is items
    assert items == [1, 100000, 21, 30, 4]
```

Reply:
done